spec/factories/auth_source_factory.rb

```diff
@@ -36,4 +36,8 @@
     port 225  # a reserved port, should not be in use
     attr_login 'uid'
   end
+
+  factory :dummy_auth_source, class: DummyAuthSource do
+    name 'DerpLAP'
```

Comment:
I like the dummy auth source, though. I needed it for testing the use case that there is an LDAP auth source without actually having a LDAP server available. I know we have an LDAP in the company but it's just a pain because I have to lookup the data and have to connect to the VPN and stuff. Also I can only login myself as opposed to random users for testing.

Considering this and the fact that random OpenSource developers, who may want to work on OpenProject, probably don't have an LDAP either, I suggest we keep this around to make everyone's life a little bit easier.